server-timing/object-cache.copy.php

```diff
@@ -1,5 +1,14 @@
 <?php
 /**
+ * Plugin Name: Performance Lab Server Timing Object Cache Drop-In
+ * Plugin URI: https://github.com/WordPress/performance
+ * Description: Performance Lab drop-in to register Server-Timing metrics early. This is not a real object cache drop-in and will not override other actual object cache drop-ins.
+ * Version: 1
```

Comment:
```suggestion
 * Version: 1.0
```
Nit-pick: Can we use proper version number here?

Reply:
@mukeshpanchal27 I used that since this version is already used in the constant in this file, so we shouldn't diverge from that.

In the future, if we make (real) changes to this file, we can update the version number to e.g. `2.0.0`, but I think at this point it's not worth doing.

Part of me also thinks it's worth using a different versioning approach here, since this file needs to be copied over, and we shouldn't do that unless necessary. So we should keep the version here independent of the "main" version of the overall plugin, and by using another versioning approach that becomes obvious.

Reply:
👍